streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -1553,68 +1553,16 @@ public void shouldRecoverFromInvalidOffsetExceptionOnRestoreAndFinishRestore() t
     }
 
     @Test
-    public void shouldRecordSkippedMetricForDeserializationExceptionWithBuiltInMetricsVersionLatest() {
-        final LogCaptureAppender appender = LogCaptureAppender.createAndRegister();
-
-        internalTopologyBuilder.addSource(null, "source1", null, null, null, topic1);
-
-        final Properties config = configProps(false);
-        config.setProperty(
-            StreamsConfig.DEFAULT_DESERIALIZATION_EXCEPTION_HANDLER_CLASS_CONFIG,
-            LogAndContinueExceptionHandler.class.getName()
-        );
-        config.setProperty(
-            StreamsConfig.BUILT_IN_METRICS_VERSION_CONFIG,
-            StreamsConfig.METRICS_LATEST
-        );
-        config.setProperty(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.Integer().getClass().getName());
-        final StreamThread thread = createStreamThread(CLIENT_ID, new StreamsConfig(config), false);
-
-        thread.setState(StreamThread.State.STARTING);
-        thread.setState(StreamThread.State.PARTITIONS_REVOKED);
-
-        final TaskId task1 = new TaskId(0, t1p1.partition());
-        final Set<TopicPartition> assignedPartitions = Collections.singleton(t1p1);
-        thread.taskManager().setPartitionsToTaskId(Collections.singletonMap(t1p1, task1));
-        thread.taskManager().setAssignmentMetadata(
-            Collections.singletonMap(task1, assignedPartitions),
-            Collections.emptyMap());
-
-        final MockConsumer<byte[], byte[]> mockConsumer = (MockConsumer<byte[], byte[]>) thread.consumer;
-        mockConsumer.assign(Collections.singleton(t1p1));
-        mockConsumer.updateBeginningOffsets(Collections.singletonMap(t1p1, 0L));
-        thread.rebalanceListener.onPartitionsAssigned(assignedPartitions);
-        thread.runOnce();
-
-        long offset = -1;
-        mockConsumer.addRecord(new ConsumerRecord<>(
-            t1p1.topic(),
-            t1p1.partition(),
-            ++offset, -1,
-            TimestampType.CREATE_TIME,
-            ConsumerRecord.NULL_CHECKSUM,
-            -1,
-            -1,
-            new byte[0],
-            "I am not an integer.".getBytes()));
-        mockConsumer.addRecord(new ConsumerRecord<>(
-            t1p1.topic(),
-            t1p1.partition(),
-            ++offset,
-            -1,
-            TimestampType.CREATE_TIME,
-            ConsumerRecord.NULL_CHECKSUM,
-            -1,
-            -1,
-            new byte[0],
-            "I am not an integer.".getBytes()));
-        thread.runOnce();
-
-        verifyLogMessagesSkippedRecordsForDeserializationException(appender);
+    public void shouldLogAndRecordSkippedMetricForDeserializationExceptionWithBuiltInMetricsVersionLatest() {
```

Comment:
What about the other `shouldReportSkippedRecordsForInvalidTimestampsWithBuiltInMetricsVersionLatest` and `shouldReportSkippedRecordsForInvalidTimestampsWithBuiltInMetricsVersion0100To24`?

Reply:
Ouch! Missed them, sorry. Will do.